.github/workflows/image.yaml

```diff
@@ -0,0 +1,179 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this workflow on pull requests or merge to main/release branches
+name: image
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+      - release-*
+  push:
+    branches:
+      - main
+      - release-*
+
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        driver:
+          - 470.223.02
+          - 535.161.07
+          - 550.54.14
+        dist:
+          - ubuntu20.04
+          - ubuntu22.04
+          - rhel8
+          - centos7
+          - rhel9
+          - fedora36
+        ispr:
+          - ${{github.event_name == 'pull_request'}}
+        exclude:
+          - ispr: true
+            dist: fedora36
+          - ispr: true
+            dist: rhel8
+          - ispr: true
+            dist: rhel8
```

Comment:
If I made a change to the `rhel8` folder in a PR, is there anyway to trigger the build for it?

Reply:
That's a good question. I think we could have excludes based on changed patterns.

Reply:
@ArangoGutierrez why was this question resolved?

Reply:
it should be a follow up